.github/workflows/wpt.yml

```diff
@@ -49,11 +49,14 @@ jobs:
       - name: Set up hosts
         run: ./wpt make-hosts-file | sudo tee -a /etc/hosts
         working-directory: wpt
-      - name: Install Chromium
-        shell: bash # force -eo pipefail
-        run: |
-          chromium_binary="$(npx @puppeteer/browsers install chromium@latest | cut -f 2- -d' ')"
-          echo "chromium_binary=$chromium_binary" >> $GITHUB_ENV
+      - name: Setup cache for browser binaries
+        uses: actions/cache@v3
+        with:
+          path: ~/.cache/chromium-bidi
```

Comment:
If the name of the action is browser agnostic, let's make the cache path browser agnostic as well.

`~/.cache/browsers` or similar

Reply:
`chromium-bidi` is the name of the repository though to avoid conflicts with other cashes.